code/game/objects/structures.dm

```diff
@@ -1,12 +1,30 @@
 /obj/structure
 	icon = 'icons/obj/structures.dmi'
 	penetration_dampening = 5
+	var/hasbolts = FALSE
+
+/obj/structure/examine(mob/user)
+	..()
+	if(hasbolts)
+		to_chat(user,"<span class='info'>This one is bolted into place.</span>")
 
 /obj/structure/blob_act(var/destroy = 0)
 	..()
 	if(destroy || (prob(50)))
 		qdel(src)
 
+/obj/structure/attackby(obj/item/I, mob/user)
```

Comment:
nothing personal, `/obj/structure/bed/chair/vehicle/janicart`

Reply:
I believe vehicles are anchored, but also this would be funny

Reply:
You are correct, vehicles are already anchored so they can't be bolted. I do agree that bolting a vehicle would be very funny though.

I think your comment has an underlying insight that testing this code is going to be a bitch given the number of structures but I am going to do my best (when it's ready).

Reply:
i skimmed through the list and the only thing that caught my eye is the giftwrap strange present

Reply:
What happens if you bolt the singulo? Or Narsie? They're structures aren't they?

Reply:
They're machinery, and also anchored.

Reply:
Singulos pulling stuff unanchors them by default, right? Can you unanchor a singulo via this process and bolt it?

Reply:
They're machinery, not structures.

Reply:
oh, aight